src/oci-umount.c

```diff
@@ -905,7 +896,7 @@ char *getJSONstring(FILE *from, size_t chunksize, char *msg)
 {
 	struct stat stat_buf;
 	char *err = NULL, *JSONstring = NULL;
-	size_t nbytes, bufsize;
+    size_t nbytes, bufsize;
```

Comment:
Fix tabs to match current behaviour

Reply:
ack, will push new commit with tabbing fixed. Thanks